Kernel/TeukolskySource.m

```diff
@@ -73,7 +73,10 @@
          |>;
 
   TeukolskySourceObject[assoc]
-]
+](*/;(orbit[e]==0 && Abs[orbit["Inclination"]]==1);
```

Comment:
Is there a reason for adding this commented out code?

Reply:
No I hadn't realised I had made any changes to that file. Should be removed now.